### PR TITLE
making base api working with local_api and export baseUrl (without /api)

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

File: BaseApi.js [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
 import * as Device from 'expo-device';
 
-const baseUrl = (() => {
+export const baseUrl = (() => {
   /*global __DEV__*/
   if (__DEV__) {
     if (Constants.manifest.extra.localAPI) {
-      if (Device.isDevice) {
-        return 'http://localhost:8004/api';
+      if (Device.isDevice && !Device.brand === 'google') {
```
Comment (goapunk) [inline BaseApi.js:8]: probably missed it but why do we need this?

Comment (goapunk) [reply BaseApi.js:8]: but why would you start it with `localAPI=True` on a real phone? I think I added this case for the web (which was detected as device). On a real device the url would have to be the one of your laptop running local aplus.

Comment (khamui) [reply BaseApi.js:8]: strange, because when running in the emulator isDevice is true too?

Comment (khamui) [reply BaseApi.js:8]: Yes sound like a better workaround, will change and test it! thanks for checking.

**Comment by goapunk** [comment]
cool!

**Pull request merged**